.github/workflows/build_windows.yml

```diff
@@ -62,7 +62,11 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: build
-        run: cargo tauri build
+        run: |
+          cd mirrorx
+          yarn install
+          cd ..
+          cargo tauri build
```

Comment:
It should be add to tauri.conf.json or package.json to make all platform available